### ISSUE TITLE
Download bar

### ISSUE THREAD

**Issue by speed2CZ** [FAForever/downlords-faf-client#224]
the bar that should be showing progress of file that is being downloaded is at 0% and then flashes to 100% when file gets downloaded. Then moves to another file... same story.

Seen this will all the files when downloading 3652 patch

**Comment by micheljung** [comment]
I am not exactly sure what you wanted to tell me... :-) Could you rephrase?

**Comment by Stupinator** [comment]
he means it doesnt show progress during download between starting at 0 and finishing at 100.
